Prog/Updates/Update_Sec8mf_2017_06.sas

```diff
@@ -0,0 +1,24 @@
+/**************************************************************************
```

Comment:
There was no June update file for Sec8MF, right? If so, then you should just delete this program.